src/Cards.h

```diff
@@ -0,0 +1,112 @@
+
+#pragma once
+#ifndef EIGHT_MINUTE_EMPIRE_CARDS_H
+#define EIGHT_MINUTE_EMPIRE_CARDS_H
+
+#include <string>
+#include <ostream>
+#include <vector>
+#include <ctime>
+#include <algorithm>
+#include <iostream>
+using namespace std;
+
+using std::string;
+
+class Cards {
+public:
+    Cards();
+
+    Cards(const string &good, const string &action);
+
+    virtual ~Cards();
+
+    const string &getGood() const;
+
+    void setGood(const string &good);
+
+    const string &getAction() const;
+
+    void setAction(const string &action);
+
+    friend ostream &operator<<(ostream &os, const Cards &cards);
+
+public:
+
+
+
+
+private:
+    string good;
+    string action;
+};
+
+class Deck {
+    std::vector<Cards> deck;
+
+public:
```

Comment:
@mdesson Do you think this is how we should be organizing card data? I think its okay for now but is there a way to parse these strings to determine action types down the road? Also, shouldn't these cards be private data members? And if so... do you think we will get punished for not having them as pointers.

Reply:
>  Do you think this is how we should be organizing card data? I think its okay for now but is there a way to parse these strings to determine action types down the road?

The way they're set up now, it's going to be a lot of string processing down the road. Assignment 1 is to get a structure going, but we can change it in assignment 2 into something a little more organized.

Maybe (and this is a rough attempt): `Cards(resourceName, resourceQuantity, actionType1, actionQuantity1, actionType2, actionQuantity2)`

The game itself will processing this, though with the data passed to your Player case, so we have some wiggle room. We could also do a nested data structure like, but let's avoid that if we can.